git/git.go

```diff
@@ -1,40 +1,23 @@
 package git
 
 import (
-	"bufio"
 	"fmt"
 	"os/exec"
+	"strings"
 )
 
 var execCommand = exec.Command
 
 // Tags returns the list of git tags as a string slice
 func Tags() ([]string, error) {
 	cmd := execCommand("git", "tag")
-	outReader, err := cmd.StdoutPipe()
+	out, err := cmd.Output()
```

Comment:
thorscmversion is super inefficient for each tag. Thousands of tags will only be improved by doing filtering with something like a --baseversion flag, and at that point we will probably need to switch back to streaming the output. We can tackle that when we get to it.